src/web/inspectors/payload/PayloadAccordionContext.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2024 Kapeta Inc.
+ * SPDX-License-Identifier: MIT
+ */
+
+import React, { ReactNode, useMemo, useState } from 'react';
+
+export interface PayloadAccordionContextType {
+    /**
+     * If the raw preview is enabled
+     */
+    raw: boolean;
+    /**
+     * Callback when the raw preview is toggled
+     */
+    setRaw: (raw: boolean) => void;
+}
+
+const PayloadAccordionContext = React.createContext<PayloadAccordionContextType | undefined>(undefined);
```

Comment:
Seems like a very simple thing to create a context for, what does it buy us / is the component tree complex since you chose this?